wpilib/wpilib/speedcontrollergroup.py

```diff
@@ -26,15 +26,12 @@ def __init__(self, speedController, *args):
         SendableBase.__init__(self)
         SpeedController.__init__(self)
 
-        self.speedControllers = []
-        self.speedControllers.append(speedController)
+        self.speedControllers = [speedController] + list(args)
         self.addChild(speedController)
 
         for speedcontroller in args:
```

Comment:
Change this to `for speedcontroller in self.speedControllers` and you won't need the extra `self.addChild()` above.

Reply:
o.O